test/RulesTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RulesTest {
+
+    private RockRule rockRule;
+
+    @Before
+    public void setUp() throws Exception {
+        rockRule = new RockRule();
+    }
+
+    @Test
+    public void rockNotOneOfPlayersMove() {
+        assertEquals(null, rockRule.decideWinner(SYMBOL.PAPER, SYMBOL.PAPER));
+    }
+
+    @Test
+    public void rockBeatsScissors1() {
```

Comment:
This test name smelled to me (1 and 2). Is the behaviour that we want to be getting the symbol that won? Or is it, given one person's choice and another person's choice, who won the Roshambo? If we change the answer to our question, these tests can probably be changed to not be smelly with the *1 and *2 suffix.